sgf-viewer/src/content.ts

```diff
@@ -179,6 +179,24 @@ export const pages: object =
                 "adversary": "498 million training steps, 600 visits",
                 "victim": "Tony Wang (Author)",
                 "description": ["The first-author of this paper (a Go novice) also managed to beat the adversary that was trained against our hardened victim, which had a pass-alive defense. The author was still able to win, demonstrating that this alternate version of our adversary is still no stronger than a Go amateur."]
+            },
+            {
+                "title": "Human analysis of hardened victim vs. adversary",
+                "dir_name": "analysis",
+                "server": "dqn.ist.berkeley.edu",
+                "paths": [
+                    "/nas/ucb/josephmiller/sgf-viewer-games/kellin-analysis-game.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "498 million training steps, 600 visits",
+                "victim": "Latest, 1600 visits, pass-alive defense",
+                "description": [
+                    "An expert-level (6 dan) human player on our team analyzed the following game. It shows typical behavior and outcomes with an adversary trained on and playing a hardened KataGo victim: the victim gains an early and soon seemingly insurmountable lead. The adversary sets a trap that would be easy for a human to see and avoid. But the victim is oblivious and collapses.",
```

Comment:
Nit for @UFO-101 : The spacing between paragraphs on the resulting webpage is large and looks unnatural

Reply:
Fixed